File: ibm/src/dotnet/Custom/Custom.cs
using System;
using Newtonsoft.Json.Linq;
using System.Diagnostics;

namespace Custom
{
    public class CustomDotnet
    {
        public JObject Main(JObject args)
        {
            Stopwatch sw = new Stopwatch();
            sw.Start();

            /* 
            TODO: put your code here
            You can basically do anything you want,
            but please leave the return statement header
            and the success field as it is.
            */

            sw.Stop();
            JObject message = new JObject();
            message.Add("success", new JValue(true));
            JObject payload = new JObject();
            payload.Add("test", new JValue("custom test"));
            payload.Add("time", new JValue(sw.Elapsed.TotalMilliseconds));
            message.Add("payload", payload);

            return (message);
        }
    }
 }

File: ibm/src/go/custom/custom.go
package main

import (
    "time"
)

func Main(params map[string]interface{}) map[string]interface{} {
    start := time.Now()

    /* 
    TODO: put your code here
    You can basically do anything you want,
    but please leave the return statement header
    and the success field as it is.
    */

    elapsed := time.Since(start)
    msg := map[string]interface{}{}
    msg["success"] = true
    msg["payload"] = map[string]interface{}{}
    msg["payload"].(map[string]interface{})["test"] = "custom test"
    msg["payload"].(map[string]interface{})["time"] = int64(elapsed / time.Millisecond)

    return msg
}
